### ISSUE TITLE
Make port of Musicbot Fix

### ISSUE THREAD

**Issue by lam0r86** [Just-Some-Bots/MusicBot#1329]
Hey Guys, 
got a Firewall blocking Output of UDP/TCP
Is there a way to make the Port of the Musicbot fix?
Now its random, and i dont want to open all ports of my Rapsberrypi

Greetz

**Comment by TheBrambleShark** [comment]
Discord API traffic is handled over port 443, which is the port which the traffic leaves your computer as well as the port where it arrives at Discord's servers. When you reach out to the server, the server replies with a port to continue your communication. This port is chosen randomly from a range from about 1400+ (I don't remember the exact number off the top of my head).

**However**

In order for your computer to handle the port assignment (say the Discord API server says it wants you to use 25565), then *outbound* traffic *must* be allowed out of 25565 on your machine. Egress filtering is strongly encouraged, especially for Enterprise environments, but said egress filtering is handled at the edge firewall, not at the workstation. You can block outbound ports on a workstation, but it's best handled as a blacklist (closing specific ports) instead of a whitelist. If you whitelist, you will run into all kinds of access issues.

**TL;DR**

Don't do egress filtering at the device level. Allow all (`*`) outbound traffic and apply egress filtering at the edge router if that is a requirement.
